x/ecocredit/base/types/v1/features/msg_add_allowed_bridge_chain.feature

```diff
@@ -37,4 +37,18 @@ Feature: MsgAddAllowedBridgeChain
     }
     """
     When the message is validated
-    Then expect the error "chain_name cannot be empty: invalid request"
\ No newline at end of file
+    Then expect the error "chain_name cannot be empty: invalid request"
+  
+  Scenario: a valid amino message
+    Given the message
+    """
+    {
+      "authority": "regen1depk54cuajgkzea6zpgkq36tnjwdzv4ak663u6",
+      "chain_name": "polygon"
+    }
+    """
+    When message sign bytes queried
+    Then expect the sign bytes
+    """
+    {"type":"regen/MsgAddAllowedBridgeChain","value":{"authority":"regen1depk54cuajgkzea6zpgkq36tnjwdzv4ak663u6","chain_name":"polygon"}}
```

Comment:
Can we format this so that each field is a new line (i.e. same as other json objects in feature files)?